src/services/user-service.js

```diff
@@ -0,0 +1,80 @@
+import { Users, TokenRequests } from '../models';
+import Config from '../config/config';
+import { updateHourDayWeakData } from './token-limit-service'; 
```

Comment:
Typo: updateHourDayWeakData
```suggestion
import { updateHourDayWeekData } from './token-limit-service'; 
```

Please fix in the entire project

Reply:
Multiple places 'week' has been misspelled. I will fix all those.